less/mofo-ui-overrides.less

```diff
@@ -0,0 +1,84 @@
+.mui-tab-switcher {
+  padding-bottom: 50px;
+  position: relative;
+  padding-top: 30px;
+
+  .panel {
+    box-shadow: none;
+  }
+
+  button.btn, button.btn:active, button.btn:hover {
```

Comment:
Generally we should avoid naming elements in selectors. Can this just be `.btn, .btn:active, .btn:hover`?